PlayerTwo/canvas.js

```diff
@@ -13,100 +13,76 @@ window.addEventListener("load", () => {
     let shouldClearCanvas = false;
 
 
-    function startPosition(e) {
+    function startPosition() {
         drawing = true;
-        draw(e);
     }
     function endPosition() {
         drawing = false;
         ctx.beginPath();
     }
-    function draw(e) {
+    function draw(x, y) {
         if (!drawing) return;
 
         if (brush) { // moved this up so that this could possibly also change stroke size when using the eraser?
             ctx.strokeStyle = "black"
         } else {
             ctx.strokeStyle = "white"
         }
-        ctx.lineWidth = 2.5;
+        ctx.lineWidth = 5;
         ctx.lineCap = "round";
 
         // get position of the mouse on the canvas
-        var pos = getMousePos(e);
 
-        ctx.lineTo(pos.x, pos.y);
-        ctx.stroke();
         ctx.beginPath();
-        ctx.moveTo(pos.x, pos.y);
-    }
-
-    function switchToErase() {
-        brush = false;
-    }
-
-    function switchToBrush() {
-        brush = true;
+        ctx.lineTo(x,y);
+        ctx.stroke();
+        ctx.moveTo(x, y);
```

Comment:
I like how you decided to simplify this function, rather than try to do exactly the same as player one! I think we might need to set up a path for the next line at the end, so we can connect the lines drawn when the player is moving their mouse too quickly :)